### PR TITLE
fixing ecr url

### PR DESCRIPTION
**Pull request by serventera**

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -134,25 +134,25 @@ jobs:
...
+      with:
+        context: .
+        # Android SDK not support arm64 yet
+        # platforms: linux/amd64,linux/arm64
+        platforms: linux/amd64
+        # platforms: linux/arm64
+        # push: ${{ github.event_name != 'pull_request' && github.ref == 'refs/heads/main' }}
+        push: ${{ github.event_name != 'pull_request' && (github.ref == 'refs/heads/main' || contains(github.ref, 'refs/tags/')) }}
+        tags: public.ecr.aws/r4f2m1l8/android:latest,${{ steps.meta.outputs.tags }}
+        labels: ${{ steps.meta.outputs.labels }}
```
Comment (mochadwi) [inline .github/workflows/docker-image.yml:158]: mas @serventera ini docker github action-nya bisa ngerti mau push ke ECR langsung kah dengan tags yg line 157?

**Review by mochadwi — COMMENTED** [review]
lgtm mas, but curious about `docker/build-push-action` behavior is it push to `hub.docker.com` or able to push onto Sampingan ECR with `tags` attributes cmiiw

File: .github/workflows/docker-image.yml (hunk 1 above)
Comment (serventera) [reply .github/workflows/docker-image.yml:158]: iya harusnya ngerti bal

File: .github/workflows/docker-image.yml [hunk 2]
```diff
@@ -134,25 +134,25 @@ jobs:
       id: login-ecr
       uses: aws-actions/amazon-ecr-login@v1
```
Comment (mochadwi) [inline .github/workflows/docker-image.yml:135]: Mas, as `docker/build-push-action` able to push onto ECR, does it safe to remove ECR Credentials attributes? or is it still necessary? @serventera

File: .github/workflows/docker-image.yml (hunk 1 above)
Comment (mochadwi) [reply .github/workflows/docker-image.yml:158]: naiisuuu thanks mas.

curious with this part:
Screen Shot 2022-03-08 at 6 14 30 PM

different when targeting image url to `hub.docker` as below:

Screen Shot 2022-03-08 at 6 15 44 PM

Comment (mochadwi) [reply .github/workflows/docker-image.yml:158]: hopefully inshaAllah no need to adjust anything in the Gitlab CI 🙏 after merged

**Review by sakti — APPROVED** [review]
LGTM

File: .github/workflows/docker-image.yml (hunk 2 above)
Comment (serventera) [reply .github/workflows/docker-image.yml:135]: masih perlu bal buat login ke AWS nya. soale klo g login nanti g bisa push ke ECR nya.

File: .github/workflows/docker-image.yml (hunk 1 above)
Comment (mochadwi) [reply .github/workflows/docker-image.yml:158]: notes:

yg former: itu ga masked (di CI build-nya)

yg later: itu masked (di CI build-nya)

**Pull request merged**